src/metabase/sync/sync_metadata/fields.clj

```diff
@@ -264,14 +337,23 @@
   ([database :- i/DatabaseInstance, table :- i/TableInstance]
    (sync-util/with-error-handling (format "Error syncing fields for %s" (sync-util/name-for-logging table))
      (let [db-metadata (db-metadata database table)]
+
+       (update! fields-per-table-hist (count db-metadata))
+
        ;; make sure the instances of Field are in-sync
-       (sync-field-instances! table db-metadata (our-metadata table) nil)
+       (time! sync-fields-timer
```

Comment:
On average, how long does it take us to sync fields?

![sync-fields-timer](https://user-images.githubusercontent.com/215910/34177229-81dbfc32-e4c8-11e7-8894-214200f96e85.png)